scripts/token-create/create-token.go

```diff
@@ -0,0 +1,132 @@
+	accountID := flag.String("accountId", "0.0", "Hedera Account ID")
+	network := flag.String("network", "", "Hedera Network Type")
+	bridgeID := flag.String("bridgeID", "0.0", "Bridge account ID")
+	memberPrKeys := flag.String("memberPrKeys", "", "The count of the members")
+	flag.Parse()
+	if *privateKey == "0x0" {
+		panic("Private key was not provided")
+	}
+	if *accountID == "0.0" {
+		panic("Account id was not provided")
+	}
+	if *bridgeID == "0.0" {
+		panic("Bridge id was not provided")
+	}
+
+	fmt.Println("-----------Start-----------")
+	client := initClient(*privateKey, *accountID, *network)
+
+	membersSlice := strings.Split(*memberPrKeys, " ")
+
+	var custodianKey []hedera.PrivateKey
+	for i := 0; i < len(membersSlice); i++ {
+		privateKeyFromStr, err := hedera.PrivateKeyFromString(membersSlice[i])
+		if err != nil {
+			panic(err)
+		}
+		custodianKey = append(custodianKey, privateKeyFromStr)
+	}
+
+	tokenId := createToken(client)
+
+	bridgeIDFromString, err := hedera.AccountIDFromString(*bridgeID)
+	if err != nil {
+		panic(err)
+	}
+
+	receipt := associateTokenToAccount(client, *tokenId, bridgeIDFromString, custodianKey)
+	fmt.Println("Token ID:", tokenId)
+	fmt.Println("Associate transaction status:", receipt.Status)
+}
+func createToken(client *hedera.Client) *hedera.TokenID {
+	result, err := hedera.NewTokenCreateTransaction().
+		SetTreasuryAccountID(client.GetOperatorAccountID()).
+		SetTokenName("e2e-test-token").
+		SetTokenSymbol("ett").
+		SetInitialSupply(100000000000000).
+		SetDecimals(8).
+		SetMaxTransactionFee(hedera.HbarFrom(20, "hbar")).
+		Execute(client)
+	if err != nil {
+		fmt.Println(err)
+	}
+	receipt, err := result.GetReceipt(client)
+	if err != nil {
+		fmt.Println(err)
+	}
+	return receipt.TokenID
+}
+func associateTokenToAccount(client *hedera.Client, token hedera.TokenID, bridgeID hedera.AccountID, custodianKey []hedera.PrivateKey) hedera.TransactionReceipt {
+	res, err := hedera.
+		NewTokenAssociateTransaction().
+		SetAccountID(bridgeID).
+		SetTokenIDs(token).
+		FreezeWith(client)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	for i := 0; i < len(custodianKey); i++ {
+		res = res.Sign(custodianKey[i])
+	}
+	res2, err := res.Execute(client)
+	if err != nil {
+		fmt.Println(err)
+	}
+	receipt, err := res2.GetReceipt(client)
+	if err != nil {
+		fmt.Println(err)
+	}
+	return receipt
+}
+func initClient(privateKey, accountID, network string) *hedera.Client {
+	var client *hedera.Client
+
+	if network == "previewnet" {
+		client = hedera.ClientForPreviewnet()
+	} else if network == "testnet" {
+		client = hedera.ClientForTestnet()
+	} else if network == "mainnet" {
+		client = hedera.ClientForMainnet()
+	} else {
+		panic("Unknown Network Type!")
+	}
+	accID, err := hedera.AccountIDFromString(accountID)
+	if err != nil {
+		panic(err)
+	}
+	pK, err := hedera.PrivateKeyFromString(privateKey)
+	if err != nil {
+		panic(err)
+	}
+	client.SetOperator(accID, pK)
+	return client
+}
```

Comment:
This code is duplicated a couple of times.
- `create-token`
- `associate-token`
- `hedera-setup`

This code is used in the e2e tests as-well.

At minimum, we should introduce `client.go` in the scripts folder f.e and reuse it in the different scripts